cfn.properties

```diff
@@ -0,0 +1,14 @@
+name=Cloudformation Plugin for SonarQube
+category=External Analysers
+descripton=Cloudformation rules for json and yaml, supporting cfn-nag
+homepageUrl=https://github.com/Hack23/sonar-cloudformation-plugin
+publicVersions=2.0.1
+
+defaults.mavenGroupId=com.hack23.sonar
+defaults.mavenArtifactId=sonar-cloudformation-plugin
+
+2.0.1.description=Initial Marketplace entry
+2.0.1.date=2020-03-24
+2.0.1.sqVersions=[7.7,LATEST]
```

Comment:
Please update this range to start at 7.9, which is the oldest version still supported in the Marketplace.

Reply:
Updated to 7.9